curriculum/challenges/english/25-front-end-development/quiz-tooling-and-deployment/66fd6f0370213be90fed82ec.md

```diff
@@ -17,439 +17,438 @@ Answer all of the questions below correctly to pass the quiz.
+What is the function of the post method in a form?
 
 #### --distractors--
 
-Placeholder distractor 1
+For updating data
 
 ---
 
-Placeholder distractor 2
+For deleting data
 
 ---
 
-Placeholder distractor 3
+For reading data
 
 #### --answer--
 
-Placeholder answer
+For creating data
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the primary purpose of an identity provider?
 
 #### --distractors--
 
-Placeholder distractor 1
+To store user data
 
 ---
 
-Placeholder distractor 2
+To process payments
 
 ---
 
-Placeholder distractor 3
+To host a website
 
 #### --answer--
 
-Placeholder answer
+To authenticate users
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Why is it important to integrate an identity provider in mordern application?
 
 #### --distractors--
 
-Placeholder distractor 1
+It simplifies the process of creating a website
 
 ---
 
-Placeholder distractor 2
+It speeds up frontend performance
 
 ---
 
-Placeholder distractor 3
+It simplifies database management
 
 #### --answer--
 
-Placeholder answer
+It provides a secure way to authenticate users
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+In addition to authentictication, what other feature do identity providers offer?
 
 #### --distractors--
 
-Placeholder distractor 1
+Database management
 
 ---
 
-Placeholder distractor 2
+User interface design
 
 ---
 
-Placeholder distractor 3
+Content delivery
 
 #### --answer--
 
-Placeholder answer
+Authorization control
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following platforms provides a built-in continuous deployment feature?
 
 #### --distractors--
 
-Placeholder distractor 1
+AWS EC2
 
 ---
 
-Placeholder distractor 2
+GitHub Pages
 
 ---
 
-Placeholder distractor 3
+VPS
 
 #### --answer--
 
-Placeholder answer
+Netlify
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which deployment platform is optimized for front-end projects like static websites?
 
 #### --distractors--
 
-Placeholder distractor 1
+AWS Lambda
 
 ---
 
-Placeholder distractor 2
+VPS
 
 ---
 
-Placeholder distractor 3
+Heroku
 
 #### --answer--
 
-Placeholder answer
+Vercel
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which type of server hosting allows you to fully control the server, including the operating system?
 
 #### --distractors--
 
-Placeholder distractor 1
+PaaS (Platform as a Service)
 
 ---
 
-Placeholder distractor 2
+Serverless functions
 
 ---
 
-Placeholder distractor 3
+SaaS (Software as a Service)
 
 #### --answer--
 
-Placeholder answer
+VPS (Virtual Private Server)
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which deployment option would you choose for full control over your deployment environment?
 
 #### --distractors--
 
-Placeholder distractor 1
+Vercel
 
 ---
 
-Placeholder distractor 2
+Netlify
 
 ---
 
-Placeholder distractor 3
+GitHub Pages
 
 #### --answer--
 
-Placeholder answer
-
+VPS
```

Comment:
```suggestion
VPS

```

Restore the EOF new line.

Reply:
resolving the conversation doesn't mean that the issue is fixed, please actually do the thing @SelormDev

Reply:
Pardon me but I thought commiting the suggestion will apply the changes requested. Let me manually check and do the necessary changes.

Reply:
the diff still shows a removed line

![image](https://github.com/user-attachments/assets/608c00e4-2338-45fc-8624-ffe47e2a5cfc)

Reply:
now it's fine, I committed the suggestion